bbb_presentation_video/renderer/tldraw/utils.py

```diff
@@ -349,3 +487,178 @@ def draw_smooth_stroke_point_path(
 ) -> None:
     outline_points = list(map(lambda p: p["point"], points))
     draw_smooth_path(ctx, outline_points, closed)
+
+
+def pattern_fill(fill: Color, opacity: float = 1) -> cairo.SurfacePattern:
+    surface = cairo.ImageSurface(cairo.FORMAT_ARGB32, 8, 8)
+    ctx = cairo.Context(surface)
+
+    # RGB for #fcfffe
+    ctx.set_source_rgba(252 / 255, 255 / 255, 254 / 255, opacity)
```

Comment:
I'd appreciate pulling this color out to a constant at the top of the file, like CANVAS, STICKY_TEXT_COLOR, etc.